lib/pynput/keyboard/_darwin.py

```diff
@@ -34,17 +34,42 @@
     ListenerMixin)
 from . import _base
 
-
-# From NSEvent.h
-NSSystemDefined = 14
-
-# From hidsystem/ev_keymap.h
-NX_KEYTYPE_PLAY = 16
-NX_KEYTYPE_MUTE = 7
-NX_KEYTYPE_SOUND_DOWN = 1
-NX_KEYTYPE_SOUND_UP = 0
-NX_KEYTYPE_NEXT = 17
-NX_KEYTYPE_PREVIOUS = 18
+'''
+#https://gist.github.com/nevyn/764542/dc1f7117a656314accb4ae206f7483fc1513180b
+#https://weblog.rogueamoeba.com/2007/09/29/apple-keyboard-media-key-event-handling/
+#https://chromium.googlesource.com/experimental/chromium/src/+/refs/wip/bajones/
+    webvr/chrome/browser/extensions/global_shortcut_listener_mac.mm
+'''
+kSystemDefinedEventMediaKeysSubtype = 8
+
+#iokit hidsystem/ev_keymap.h
+tKeyNameToCode = {
```

Comment:
I don't really see the reason for this to be a dict, so unless you manage to convince me otherwise, I will keep the old behaviour.

Reply:
Thanks for the question. In my opinion, there are a few advantages of using a dict. Example:

```
#native definition
tableNativeNameToCode = {
	'NX_KEYTYPE_SOUND_UP': 0,
	'NX_KEYTYPE_SOUND_DOWN': 1,
	'NX_KEYTYPE_BRIGHTNESS_UP': 2,
	'NX_KEYTYPE_BRIGHTNESS_DOWN': 3
}

#native names to pynput naming, 1:1 visual aid
tableNativeToPynput = {
	'NX_KEYTYPE_SOUND_UP': 'media_volume_up',
	'NX_KEYTYPE_SOUND_DOWN': 'media_volume_down',
	'NX_KEYTYPE_BRIGHTNESS_UP': 'media_brightness_up',
	'NX_KEYTYPE_BRIGHTNESS_DOWN': 'media_brightness_down'
}

#useful for key controller
tablePynputToCode = dict( zip( tableNativeToPynput.values(), tableNativeNameToCode.values() ) )

#useful for key listener
tableCodeToPynput = dict( zip( tableNativeNameToCode.values(), tableNativeToPynput.values() ) )
```

This is cleaner (not only visually but can have the declarations all together - header-like), more flexible (easier to add, remove, edit, merge) and therefore easier to maintain (can track the number of keys for testing too) 

It also allows neat usages besides the usual tablePynputToCode['media_volume_up']:

```
tablePynputToCode.get('unknown_key',None) #have a default value in case key is not found
#or
def pynputToCode( x ):
    return tablePynputToCode.get(x,None)
```

And since it is a dict, it can be iterated as well to see the keys that are supported, etc

I understand you might want to stick to the previous definitions to avoid refactoring other definitions. I hope at least sounded a bit appealing :)